Makecerts/san-extras.example

```diff
@@ -0,0 +1,7 @@
+DNS.1 = kubernetes
+DNS.2 = kubernetes.default
+DNS.3 = kubernetes.default.svc
+DNS.4 = kubernetes.default.svc.cluster.local
+DNS.5 = %(master_instance)
+IP.1 = %(master_ip)
+IP.2 = 10.0.0.1
```

Comment:
Can you make this `%(kubernetes_service_ip)` since 10.0.0.01 is the default because the default service range is 10.0.0.0/16 but not everyone uses that range.